api/python/indigo/bingo/bingo.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright (C) from 2009 to Present EPAM Systems.
+#
+# This file is part of Indigo toolkit.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ctypes import CDLL
+
+from ..indigo.indigo import Indigo
+from ..indigo.indigo_lib import IndigoLib
+from ..indigo.indigo_object import IndigoObject
+from .bingo_exception import BingoException
+from .bingo_lib import BingoLib
+from .bingo_object import BingoObject
+
+
+class Bingo:
+    def __init__(self, bingo_id: int, session: Indigo, lib: BingoLib) -> None:
+        self._id: int = bingo_id
+        self.session: Indigo = session
+        self._libraryInstance: BingoLib = lib
+        # init
+        self.version()
+
+    def __del__(self):
+        self.close()
+
+    def _lib(self) -> CDLL:
+        self.session._setSessionId()  # noqa
+        return self._libraryInstance.lib  # type: ignore
+
+    def close(self):
+        if self._id != -1:
+            IndigoLib.checkResult(
+                self._lib().bingoCloseDatabase(self._id), BingoException
+            )
+            self._id = -1
+
+    @staticmethod
+    def createDatabaseFile(indigo, path, databaseType, options=""):
```

Comment:
Please add docstrings here like it was done for Inchi. I see most methods here are self-explanatory, but it would be way more convenient to have some reference with args/returns and a short description.

Reply:
Good idea, but it's not the target of this request (it's just a refactoring).